### DISCUSSION TITLE
6.4.5

### DISCUSSION THREAD

**Discussion by Ghost-chu** [PBH-BTN/PeerBanHelper#588] [公告]
##  错误修复

* 再次修复了 BitComet 下载器在 Docker 容器中下载依赖时因提前类加载导致意外调用 GUI 代码崩溃的问题

注：我们已经收到了一些有价值的错误报告，但由于开发者身体健康原因，在短期内无法实施可靠修复，因此非致命问题/修复困难的问题将被暂时推迟。对此造成的不便，深表歉意。

出现网络问题无法下载依赖的用户，可请求其它相同版本、相同系统（如 Windows/macOS/Linux）的正常用户触发可选依赖补全后，打包 libraries。

## Docker 用户

DockerHub: `ghostchu/peerbanhelper:v6.4.5`   
阿里云国内镜像加速: `registry.cn-hangzhou.aliyuncs.com/ghostchu/peerbanhelper:v6.4.5`

由于 Portable 为人工打包，本版暂不提供 Portable 包。  
基于过往版本 Portable 替换 JAR 升级的用户，请更换 jre 文件夹中的 JRE 到 Java 23 版本以缓解 selector manager closed 的错误问题。该错误将在未来版本中通过更换 HTTP 客户端修复。届时我们可能会回调最低 Java 版本到 21 版本。

<hr /><em>This discussion was created from the release <a href='https://github.com/PBH-BTN/PeerBanHelper/releases/tag/v6.4.5'>6.4.5</a>.</em>

**Comment by kndxhz** [comment]
希望您们能够先关注一下自己的身体问题哈
